readme.md

```diff
@@ -13,23 +13,26 @@
   <a href="https://travis-ci.org/xing/hops">
     <img src="https://travis-ci.org/xing/hops.svg?branch=master">
   </a>
-  <a href="https://david-dm.org/xing/hops">
-    <img src="https://david-dm.org/xing/hops.svg">
-  </a>
 </p>
 <p>&nbsp;</p>
 
-Hops is a universal [Webpack](https://webpack.js.org) wrapper. It additionally leverages both [Babel](https://babeljs.io) and [PostCSS](http://postcss.org). On the one hand, hops is designed to simplify getting started with modern frontend tooling by providing an extensible baseline config for Webpack. On the other hand, hops aims to help with using webpack for code running on servers.
+Hops is an universal [Webpack](https://webpack.js.org) wrapper. It additionally leverages both [Babel](https://babeljs.io) and [PostCSS](http://postcss.org). On the one hand, hops is designed to simplify getting started with modern frontend tooling by providing flexible baseline configs for Webpack. On the other hand, it sets up a scalable universal ("isomorphic") development environment.
```

Comment:
This is of course absolutely subjective, but I would emphasize on the isomorphic and application framework features and put the webpack wrapper features second.